public/api/conf/1.0/application.raml

```diff
@@ -99,18 +99,95 @@ traits:
               }
 
 
-  /{balanceId}:
-    uriParameters:
-      balanceId:
-        description: The ID of the balance request to check.
-        type: string
-        example: "22b9899e-24ee-48e6-a189-97d1f45391c4"
-        required: true
+  # /{balanceId}:
```

Comment:
? If this is adding the get balance by Id, why are you commenting it out in this file

Reply:
If we are not able to support multiple response codes (as per the comment above) then we will not expose this endpoint for users right away. However, it will be really useful for our testing locally.